src/components/Hero.tsx

```diff
@@ -120,16 +105,14 @@ const Hero = () => {
                   <p className="text-gray-500 text-sm font-semibold">Repository</p>
                 </div>
 
-                {fetchedData.map(data => (
+                {fetchedData.map((data) => (
                   <Combobox.Option
                     key={data.full_name}
                     as="a"
                     className={({ active }) => (active ? "bg-gray-50" : "")}
-                    value={`https://insights.opensauced.pizza/hot/repositories/filter/${data.full_name}`}
+                    value={`https://insights.opensauced.pizza/${data.name}/dashboard/filter/${data.full_name}`}
```

Comment:
We might as well change this to use the updated `app.opensauced.pizza`: we do have a DNS redirect setup, but better to not rely on that in this app code.

Reply:
Understood, I will come back with linting error fixes